draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -119,6 +121,44 @@ public void setClickToMinimizeEnabled(boolean enableClickToMinimize) {
   }
 
   /**
+   * Return if touch listener is enable or disable
+   */
+  public boolean isEnableTouchListener() {
+    return this.enableTouchListener;
+  }
+
+  /**
+   * Enable or disable the touch listener
+   *
+   * @param enableTouchListener to enable or disable the touch event.
+   */
+  public void setEnableTouchListener(boolean enableTouchListener) {
+    this.enableTouchListener = enableTouchListener;
+  }
+
+  /**
+   * Slide the view based on scroll of the nav drawer.
+   * "setEnableTouchListener" user prevents click to expand while the drawer is moving.
+   * WHen the slideOffset is bigger than 0.1 and dragView isn't close, set the dragView
+   * to minimized.
+   * It's only possible to maximize the view when @slideOffset is equals to 0.0,
+   * in other words, closed.
+   *
+   * @param slideOffset Value between 0 and 1, represent the value of slide:
+   * 0.0 is equal to close drawer and 1.0 equals open drawer.
+   * @param drawerPosition Represent the position of nav drawer on X axis.
+   * @param width Width of nav drawer
+   */
+  public void slideMinimizedView(float slideOffset, float drawerPosition, int width) {
+    float slide = Math.abs(drawerPosition);
+    if (slideOffset > 0.1 && !isClosed()) {
+      minimize();
+    }
```

Comment:
Please, remove this part of the feature. The default behavior should be just move DraggablePanel out of the window. In addition you have a bug with this minimize call. Take a look to the video -> https://drive.google.com/file/d/0B9xkpTnF9BXjR0Q3NXVWclVkOTQ/view?usp=sharing